README.md

```diff
@@ -102,7 +102,7 @@ repositories {
     mavenCentral()
 }
 dependencies {
-    annotationProcessor 'org.checkerframework:templatefora-checker:0.1-SNAPSHOT'
+    annotationProcessor 'io.github.eisop:templatefora-checker:0.1-SNAPSHOT'
```

Comment:
This should use neither `org.checkerframework` nor `io.github.eisop`. This should use an organization from the user of the template.